src/main/webapp/Portal.html

```diff
@@ -167,6 +167,10 @@
       loadCss(cdnEndpoint + 'generated/swc.css')
       loadJs(cdnEndpoint + 'generated/jquery.min.js')
       loadJs(cdnEndpoint + 'generated/jquery-ui.min.js')
+      // Load latest version of Bootstrap 3 to override the version bundled by GWTBootstrap3
+      // The Bootstrap 3 custom theme CSS is loaded through swc.scss
```

Comment:
The old CSS still looks ok after clicking around a bit after updating the JS package